packages/tracking/src/createTrackingFunction.ts

```diff
@@ -0,0 +1,71 @@
+import { getBaseProperties, type TrackEventProperties } from "./properties.ts";
+
+export interface TrackingOptions {
+    /**
+   * Whether to keep the connection alive for the tracking request.
+   * It is mostly used for tracking links where the user might navigate away before the request is completed.
+   *
+   * Caution! The body size for keepalive requests is limited to 64 kibibytes.
```

Comment:
What is a "kibibytes"?

Reply:
A kibibyte (KiB) is a unit of information or computer storage equal to 1024 bytes, used in binary systems where multiples of 2 are used instead of multiples of 10. 

https://developer.mozilla.org/en-US/docs/Web/API/RequestInit#keepalive

Reply:
haha never heard of it before!